data/tlg0527/tlg048/__cts__.xml

```diff
@@ -1,7 +1,10 @@
-<ti:work xmlns:ti="http://chs.harvard.edu/xmlns/cts" groupUrn="urn:cts:greekLit:tlg0527" xml:lang="grc" urn="urn:cts:greekLit:tlg0527.tlg048">
-  <ti:title xml:lang="eng">Isaias</ti:title>
-  <ti:edition urn="urn:cts:greekLit:tlg0527.tlg048.opp-grc2" workUrn="urn:cts:greekLit:tlg0527.tlg048">
-    <ti:label xml:lang="eng">Isaias</ti:label>
-    <ti:description xml:lang="eng">Septuaginta, Isaias</ti:description>
-  </ti:edition>
-<ti:edition urn="urn:cts:greekLit:tlg0527.tlg048.opp-eng2" workUrn="urn:cts:greekLit:tlg0527.tlg048"><ti:label xml:lang="eng">Isaias</ti:label><ti:description xml:lang="eng">Septuaginta, Isaias</ti:description></ti:edition></ti:work>
+<ti:work xmlns:cpt="http://capitains.github.io/xmlns" xmlns:dct="http://purl.org/dc/terms/" xmlns:ti="http://chs.harvard.edu/xmlns/cts" groupUrn="urn:cts:greekLit:tlg0527" xml:lang="grc" urn="urn:cts:greekLit:tlg0527.tlg048">
+  <ti:title xml:lang="lat">Isaias</ti:title>
+  <cpt:structured-metadata>
+    <dct:isVersionOf>urn:cts:hebrewLit:heb0001.heb010</dct:isVersionOf>
+  </cpt:structured-metadata>
+  <ti:translation xml:lang="eng" urn="urn:cts:greekLit:tlg0527.tlg048.opp-grc2" workUrn="urn:cts:greekLit:tlg0527.tlg048">
```

Comment:
@ahanhardt I am sorry that I keep finding problems piecemeal like this. Eventually, however, we will get this finished!
This first `<ti:translation>` element should be `<ti:edition>` instead. And it does not need an `xml:lang` attribute since it is in the same language as the original work, i.e., 'grc'.